contrib/observability_ready_util/README.md

```diff
@@ -0,0 +1,57 @@
+# OpenCensus Observability Ready Util for Java
+
+[![Build Status][travis-image]][travis-url]
+[![Windows Build Status][appveyor-image]][appveyor-url]
+
+The *OpenCensus Observability Ready Util for Java* that allows users to use OpenCensus easily.
+
+By default:
```

Comment:
Remove 'By default' and simply say 
```
It provides a wrapper that
- Enables Basic RPC Views
- Creates and Registers OCAgent Exporter to collect traces and metrics.
```
'By Defaults' suggest that there is a way to override the default but there isn't.